src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/rgw/rgw-daemon-list/rgw-daemon-list.component.html

```diff
@@ -5,10 +5,14 @@
         aria-current="page">Object Gateway</li>
   </ol>
 </nav>
+
 <cd-table [data]="daemons"
           [columns]="columns"
           columnMode="flex"
-          [detailsComponent]="detailsComponent"
-          (fetchData)="getDaemonList()"
-          [beforeShowDetails]="beforeShowDetails">
+          (updateSelection)="updateSelection($event)"
+          (fetchData)="getDaemonList()">
+
+  <cd-rgw-daemon-details cdTableDetail
```

Comment:
If this table has a detail view, you have to add `selectionType="single"` attribute to this `<cd-table>`, otherwise it will not be possible to select a row to see the row detail.